spec/csv_generator/pacbio_sample_sheet_spec.rb

```diff
@@ -38,7 +36,7 @@
 
       it 'must have the correct well header rows' do
         well_data_1 = parsed_csv[1]
-        well_data_2 = parsed_csv[7]
+        well_data_2 = parsed_csv[3] # DPL-433 why has this gone from 7 to 3 ???
```

Comment:
@BenTopping would you mind looking into why this might be with me?

Reply:
I think the issue is on line 11 where 'smrt_link_version: version10' I think this should be smrt_link_version: Pacbio::SmrtLinkVersion.find_by_name('v10') or similar

Reply:
Think it may be to do with the default pool count being changed for factory wells. https://github.com/sanger/traction-service/pull/927/files#diff-89b2e233aede7103d61c1b2d43984e443290477f795c62bf05fce653e36356d5L14

Reply:
Thanks @BenTopping 👍